<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Ground segmentation, LEFT -->
  <node name="ground_segmentation_left" pkg="linefit_ground_segmentation_ros" type="ground_segmentation_node" output="screen">
    <rosparam command="load" file="$(find linefit_ground_segmentation_ros)/launch/segmentation_params.yaml"/>
    <param name="input_topic" value="/vstl/velodyne_points" />
    <param name="ground_output_topic" value="/vstl/ground_cloud" />
    <param name="obstacle_output_topic" value="/vstl/obstacle_cloud" />
    <param name="all_points_topic" value="/vstl/all_points" />
  </node>

  <!-- Ground segmentation, RIGHT -->
  <node name="ground_segmentation_right" pkg="linefit_ground_segmentation_ros" type="ground_segmentation_node" output="screen">
    <rosparam command="load" file="$(find linefit_ground_segmentation_ros)/launch/segmentation_params.yaml"/>
    <param name="input_topic" value="/vstr/velodyne_points" />
    <param name="ground_output_topic" value="/vstr/ground_cloud" />
    <param name="obstacle_output_topic" value="/vstr/obstacle_cloud" />
    <param name="all_points_topic" value="/vstr/all_points" />
  </node>

</launch>
